x/data/server/server.go

```diff
@@ -24,12 +29,12 @@ var ModuleSchema = ormv1alpha1.ModuleSchemaDescriptor{
 
 type serverImpl struct {
 	storeKey   sdk.StoreKey
-	iriIDTable lookup.Table
+	iriHasher  hasher.Hasher
 	stateStore api.StateStore
 }
 
 func newServer(storeKey sdk.StoreKey) serverImpl {
-	tbl, err := lookup.NewTable([]byte{IriIDTablePrefix})
+	hasher, err := hasher.NewHasher([]byte{IriHashPrefix})
```

Comment:
As mentioned above, do we still need a prefix if we are using orm tables?

Reply:
no

Reply:
https://github.com/regen-network/regen-ledger/pull/970/commits/36f63e20da4bb0df865d43f342ebced627650bc5